src/djcytoscape/templates/djcytoscape/quest_map.html

```diff
@@ -11,8 +11,12 @@
 {%endif %}
 {% endblock%}
 {% block content %}
+            <p>
+              <a class="btn btn-default" href="{% url 'djcytoscape:list' %}" role="button"
+              title = "List all maps" >
+              <i class="fa fa-list-ul"></i>
+            </a>
     {% if user.is_staff %}
-        <p>
```

Comment:
Can you screenshot what these buttons look like?  For consistency, I think we should pull right and place below the heading and icon?  E.g: 
![image](https://user-images.githubusercontent.com/10604391/183523085-f6255910-e8fe-4f07-874b-0c513f29a3cf.png)

Reply:
old button layout:
![image](https://user-images.githubusercontent.com/105619909/183758997-03c0f793-7c8d-40ac-8229-cd753bb1ab69.png)
updated button layout:
![image](https://user-images.githubusercontent.com/105619909/183759256-11d94c95-cfac-4e1a-a00d-da5942108a48.png)
in the absence of an icon, the fullscreen and print buttons take up the space left behind.